examples/nomad-consul-ami/nomad-consul-docker-ecr.json

```diff
@@ -0,0 +1,159 @@
+{
+  "min_packer_version": "0.12.0",
+  "variables": {
+    "aws_region": "us-east-1",
+    "aws_profile": "default",
+    "nomad_version": "0.6.3",
+    "consul_module_version": "v0.0.3",
+    "consul_version": "0.9.3"
+  },
+  "builders": [
+    {
+      "ami_name": "nomad-consul-ubuntu-{{isotime | clean_ami_name}}",
+      "ami_description": "An Ubuntu 16.04 AMI that has Nomad and Consul installed.",
+      "instance_type": "t2.micro",
+      "name": "ubuntu16-ami",
+      "region": "{{user `aws_region`}}",
+      "profile": "{{ user `aws_profile`}}",
+      "type": "amazon-ebs",
+      "source_ami_filter": {
+        "filters": {
+          "virtualization-type": "hvm",
+          "architecture": "x86_64",
+          "name": "*ubuntu-xenial-16.04-amd64-server-*",
+          "block-device-mapping.volume-type": "gp2",
+          "root-device-type": "ebs"
+        },
+        "owners": [
+          "099720109477"
+        ],
+        "most_recent": true
+      },
+      "ssh_username": "ubuntu"
+    },
+    {
+      "ami_name": "nomad-consul-amazon-linux-{{isotime | clean_ami_name}}",
+      "ami_description": "An Amazon Linux AMI that has Nomad and Consul installed.",
```

Comment:
The names and descriptions of both of these should be updated to indicate Docker is installed too

Reply:
[Done: Name and description for AWS Linux and Ubuntu](https://github.com/MatthiasScholz/terraform-aws-nomad/commit/e4cf5a7f68310839e298672ab05b408bf463d189)